src/melee/gr/grinishie1.c

```diff
@@ -1,49 +1,239 @@
+
 #include <platform.h>
 
+#include "gr/grinishie1.h"
+
+#include "cm/camera.h"
+#include "ef/efsync.h"
+#include "gm/gm_1601.h"
+#include "gr/grdisplay.h"
+#include "gr/grlib.h"
+#include "gr/grmaterial.h"
+#include "gr/grzakogenerator.h"
 #include "gr/inlines.h"
+#include "it/it_266F.h"
+#include "it/it_26B1.h"
+#include "lb/lb_00B0.h"
+#include "lb/lb_00F9.h"
+#include "lb/lbaudio_ax.h"
+#include "mp/mplib.h"
+#include "sysdolphin/baselib/dobj.h"
+#include "sysdolphin/baselib/gobjgxlink.h"
+#include "sysdolphin/baselib/gobjproc.h"
+#include "sysdolphin/baselib/jobj.h"
+#include "sysdolphin/baselib/memory.h"
+#include "sysdolphin/baselib/random.h"
 
 #include <baselib/gobj.h>
 
+bool GetMatchTimer(u32*); // extern from gm_16AE
```

Comment:
should include this declaration from melee/gm/gm_1601.h instead of duplicating it here